tests/keras/layers/wrappers_test.py

```diff
@@ -97,7 +97,7 @@ def test_TimeDistributed_learning_phase():
     y = wrappers.TimeDistributed(core.Dropout(.999))(x, training=True)
     model = Model(x, y)
     y = model.predict(np.random.random((10, 3, 2)))
-    assert_allclose(0., y, atol=1e-1, rtol=1e-1)
+    assert_allclose(y, np.zeros(y.shape), atol=1e-1, rtol=1e-1)
```

Comment:
I think a better plan would be to test the value of the average, and use a stricter tolerance. We have a tensor of shape `(10, 3, 2)` that should be 99.9% 0s, so its average should be very close to zero.